_tests_/pet.test.js

```diff
@@ -0,0 +1,155 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+  it('returns an object', () => {
+    expect(new Pet()).toBeInstanceOf(Object);
+  });
+});
+
+describe('constructor', () => {
+  it('sets the name property', () => {
+    const pet = new Pet('fido');
+
+    expect(pet.name).toEqual('fido');
+    expect(pet.hunger).toEqual(0);
+    expect(pet.fitness).toEqual(10);
+  });
+});
+
+describe('growUp', () => {
+  it('has a initial age of 0', () => {
+    const pet = new Pet('fido');
+
+    expect(pet.age).toEqual(0);
+  });
+});
+
+describe('growUp and getting unhealthier', () => {
+  it('increases the age of our pet by 1', () => {
+    const pet = new Pet('fido');
+    pet.growUp();
+
+    expect(pet.age).toEqual(1);
+    expect(pet.hunger).toEqual(5);
+    expect(pet.fitness).toEqual(7);
+  });
+
+  it("growup does not function if pet is not alive" ,() => {
+    const pet = new Pet("fido");
+    pet.age = 30;
+    expect(() => pet.growUp()).toThrow("your pet is no longer alive :(");
+  });
+});
+
+describe('walk', () => {
+  it('increases fitness by 4 to a maximum of 10', () => {
+    const pet = new Pet('fido');
+
+    pet.fitness = 8;
+    pet.walk();
+
+    expect(pet.fitness).toEqual(10);
+  });
```

Comment:
this test only tests that `fitness` increases to a maximum of `10`, not that it increases by `4`. You should have 2 separate tests for this, one that tests it increases by `4` and one that tests the maximum is `10` e.g.
```
  it('increases fitness by 4', () => {
    const pet = new Pet('fido');

    pet.fitness = 5;
    pet.walk();

    expect(pet.fitness).toEqual(9);
  });

  it('increases fitness to a maximum of 10', () => {
    const pet = new Pet('fido');

    pet.fitness = 8;
    pet.walk();

    expect(pet.fitness).toEqual(10);
  });
```